src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
```

Comment:
Stylesheets with .scss extension cannot be linked directly in HTML. You should compile SCSS to CSS and link the compiled .css file instead.